x-pack/plugins/ml/public/application/model_management/deployment_setup.tsx

```diff
@@ -72,6 +89,69 @@ export const DeploymentSetup: FC<DeploymentSetupProps> = ({ config, onConfigChan
 
   return (
     <EuiForm component={'form'} id={'startDeploymentForm'}>
+      <EuiDescribedFormGroup
+        titleSize={'xxs'}
+        title={
+          <h3>
+            <FormattedMessage
+              id="xpack.ml.trainedModels.modelsList.startDeployment.deploymentIdLabel"
+              defaultMessage="Deployment ID"
+            />
+          </h3>
+        }
+        description={
+          <FormattedMessage
+            id="xpack.ml.trainedModels.modelsList.startDeployment.deploymentIdHelp"
+            defaultMessage="Specify unique identifier for the model deployment."
+          />
+        }
+      >
+        <EuiFormRow
+          label={
+            <FormattedMessage
+              id="xpack.ml.trainedModels.modelsList.startDeployment.deploymentIdLabel"
+              defaultMessage="Deployment ID"
+            />
+          }
+          hasChildLabel={false}
+          isInvalid={!!errors.deploymentId}
+          error={
+            <FormattedMessage
+              id="xpack.ml.trainedModels.modelsList.startDeployment.deploymentIdError"
+              defaultMessage="Deployment with this ID already exists."
+            />
+          }
+        >
+          {!isUpdate ? (
+            <EuiFieldText
```

Comment:
Is it possible to add a placeholder for the ID field, which is the model ID? Otherwise it's not clear what's going on here:

<img width="812" alt="image" src="https://user-images.githubusercontent.com/7405507/234257199-b11e36e6-2d33-4cca-ad79-ce51c2efdfce.png">